src/ui_parts/export_dialog.tscn

```diff
@@ -103,7 +102,6 @@ allow_higher = false
 unique_name_in_owner = true
 layout_mode = 2
 theme_override_colors/font_color = Color(0.666667, 0.666667, 0.666667, 1)
-theme_override_fonts/font = ExtResource("4_4t2iq")
```

Comment:
This change seems to be unintentional?

Reply:
The Godot uid bugs strike again..

Reply:
Yeah, I did not change this, Godot did it by itself.

Reply:
Alright, I reverted the changes to unrelated files.